CHANGELOG.md

```diff
@@ -163,10 +163,10 @@ New entries must be placed in a section entitled `Unreleased`.
 
 ### Editors
 
-#### Other changes
```

Comment:
We should keep this paragraph; it's in line with the rest. Or, remove it from all the other sections.